sys/include/usb.h

```diff
@@ -24,17 +24,29 @@
 extern "C" {
 #endif
 
+#include "board.h"
+
 /**
  * @defgroup usb_conf USB peripheral compile time configurations
  * @ingroup config
  * @{
  */
 
+/* These can be overridden by boards that should come up with their board
+ * supplier VID/PID pair. Boards should only override this if the RIOT built-in
+ * peripherals are compatible with whatever is usually shipped with that pair
+ * */
+#ifndef RIOT_INTERNALPERIPHERAL_VID
+/** Reserved for RIOT standard peripherals as per http://pid.codes/1209/7D00/ */
+#define RIOT_INTERNALPERIPHERAL_VID (0x1209)
+/** Reserved for RIOT standard peripherals as per http://pid.codes/1209/7D00/ */
+#define RIOT_INTERNALPERIPHERAL_PID (0x7D00)
+#endif
```

Comment:
```suggestion
#ifndef RIOT_INTERNALPERIPHERAL_VID
/** Reserved for RIOT standard peripherals as per http://pid.codes/1209/7D00/ */
#define RIOT_INTERNALPERIPHERAL_VID (0x1209)
#endif
#ifndef RIOT_INTERNALPERIPHERAL_PID
/** Reserved for RIOT standard peripherals as per http://pid.codes/1209/7D00/ */
#define RIOT_INTERNALPERIPHERAL_PID (0x7D00)
#endif
```

I'd rather have both guarded.